Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -6617,6 +6617,28 @@ These ItemGrids are available in the API and can be manipulated by the plugins,
 					},
 					Notes = "Removes one item from the hotbar's currently selected slot. Returns true on success.",
 				},
+				ReplaceOneEquippedItem =
+				{
+					Params =
+					{
+						{
+							Name = "Item",
+							Type = "cItem",
+						},
+						{
+							Name = "TryOtherSlots",
+							Type = "boolean",
+							IsOptional = true,
+						},
+					},
+					Returns =
+					{
+						{
+							Type = "number",
+						},
+					},
+					Notes = "Removes one item from the the current equipped item stack, and attempts to add the specified item stack back to the same slot. If it is not possible to place the item in the same slot, optionally (default true) tries to place the specified item elsewhere in the inventory. Returns the number of items successfully added. If the currently equipped slot is empty, this is ignored.",
```

Comment:
`this is ignored` Not true, the `Item` is still added. How about `If the currently equipped slot is empty, it is simply set to Item.` ?

Reply:
`this` was intended to be `the fact that the currently equipped slot is empty`, but I can see how this phrasing would lead to confusion. I'll change it.